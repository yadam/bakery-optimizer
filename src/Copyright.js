import React from 'react';
import Typography from '@material-ui/core/Typography';
import MuiLink from '@material-ui/core/Link';

export function Copyright() {
  return (
    <Typography variant="body2" color="textSecondary" align="center">
      {'Copyright Â© '}
      <MuiLink color="inherit" href="https://github.com/yadam">
        Adam Bantly
      </MuiLink>{' '}
      {new Date().getFullYear()}.
    </Typography>
  );
}

export default Copyright;
