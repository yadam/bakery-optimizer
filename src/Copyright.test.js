import React from 'react';
import { create } from 'react-test-renderer';
import { Copyright } from './Copyright';

describe('Breads', () => {
  it('renders', () => {
    const tree = create(<Copyright />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
