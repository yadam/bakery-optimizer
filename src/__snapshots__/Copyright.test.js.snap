// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breads renders 1`] = `
<p
  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
>
  Copyright Â© 
  <a
    className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
    href="https://github.com/yadam"
    onBlur={[Function]}
    onFocus={[Function]}
  >
    Adam Bantly
  </a>
   
  2020
  .
</p>
`;
